giskard_vision/landmark_detection/tests/report.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import pandas as pd
+
+from giskard_vision.landmark_detection.dataloaders.base import DataIteratorBase
+from giskard_vision.landmark_detection.models.base import FaceLandmarksModelBase
+from giskard_vision.landmark_detection.tests.base import Metric, Test, TestDiff
+from giskard_vision.landmark_detection.tests.performance import NMEMean
+
+
+class Report:
+    """
+    A class for generating and managing test reports for landmark detection models.
+
+    Attributes:
+        default_rel_threshold (float): Default relative threshold.
+        default_abs_threshold (float): Default absolute threshold.
+
+    """
+
+    default_rel_threshold = -0.1
```

Comment:
Why do we want this to be strictly non positive by default ? This may indicate that the test fails even when the perf is not so bad compared to ref data